components/discord_bot/actions/create-channel-invite/create-channel-invite.mjs

```diff
@@ -0,0 +1,96 @@
+import common from "../common.mjs";
+import constants from "../../constants.mjs";
+
+export default {
+  ...common,
+  key: "discord_bot-create-channel-invite",
+  name: "Create Channel Invite",
+  description: "Create a new invite for the channel. [See the docs here](https://discord.com/developers/docs/resources/channel#create-channel-invite)",
+  type: "action",
+  version: "0.0.1",
+  props: {
+    ...common.props,
+    maxAge: {
+      type: "integer",
+      label: "Max age",
+      description: "Duration of invite in seconds before expiry 0 for never. between 0 and 604800 (7 days).",
+      optional: true,
+    },
+    maxUses: {
+      type: "integer",
+      label: "Max number of uses",
+      description: "0 for unlimited. between 0 and 100.",
+      optional: true,
+    },
+    temporary: {
+      type: "boolean",
+      label: "Temporary membership",
+      description: "Whether this invite only grants temporary membership",
+      optional: true,
+    },
+    unique: {
+      type: "boolean",
+      label: "Unique",
+      description: "If true, don't try to reuse a similar invite (useful for creating many unique one time use invites)",
+      optional: true,
+    },
+    targetType: {
+      type: "integer",
+      label: "Target type",
+      description: "The type of target for this voice channel invite. [See the docs here](https://discord.com/developers/docs/resources/invite#invite-object-invite-target-types)",
+      optional: true,
+      options() {
+        return [
+          {
+            label: "Stream",
+            value: constants.INVITE_TARGET_TYPES.STREAM,
+          },
+          {
+            label: "Embedded Application",
+            value: constants.INVITE_TARGET_TYPES.EMBEDDED_APPLICATION,
+          },
+        ];
+      },
```

Comment:
Just noticed that this doesn't need to be a function. You can just return the array:
```
options: [
  {
    label: "Stream",
    value: constants.INVITE_TARGET_TYPES.STREAM,
  },
  {
    label: "Embedded Application",
    value: constants.INVITE_TARGET_TYPES.EMBEDDED_APPLICATION,
  },
]
```